### PR TITLE
Clean up org.mozilla.* in /var/folders

### PR DESCRIPTION
**Pull request by rcurranmoz**
https://bugzilla.mozilla.org/show_bug.cgi?id=1927416

Plan is to deploy this to the mac workers after work week next week

### CODE REVIEW

File: modules/worker_runner/templates/worker-runner.sh.erb [hunk 1]
```diff
@@ -20,6 +20,7 @@ rm <%= @data_dir %>/tasks-resolved-count.txt
 
 # Clear up previous task dirs
 rm -rf /opt/worker/tasks/task_*
+find /private/var/folders/ -path '*/C/org.mozilla.*' -type d -o -path '*/C/org.mozilla.*/*' -type d -exec rm -rf {} +
```
Comment (whimboo) [inline modules/worker_runner/templates/worker-runner.sh.erb:23]: Should we as well clean up the `Download` folder? Maybe a new section for both makes sense given that both are not related to the `task dirs`.

Comment (rcurranmoz) [reply modules/worker_runner/templates/worker-runner.sh.erb:23]: That's in this PR handled by a different tool https://github.com/mozilla-platform-ops/ronin_puppet/pull/774
